lib/src/main/scala/spinal/lib/bus/simple/AsyncMemoryBusFactory.scala

```diff
@@ -32,6 +32,9 @@ class AsyncMemoryBusFactory(bus: AsyncMemoryBus, incAddress: Int = 0) extends Bu
   override def readAddress()  = bus.address
   override def writeAddress() = bus.address
 
+  override def readError(): Unit = {}
+  override def writeError(): Unit = {}
```

Comment:
Ohh, I'm curious

Reply:
AsyncMemory bus doesn't have an error response mechanism. There's nothing to do during an error cycle.

Reply:
So maybe it should be an elaboration-time error to call this function for AXI, to be sure nobody misuses it?

Reply:
No as that will break from the pattern of a generic bus interface. Some buses just don't support errors.